Using `extension` on impls with associated types expects `ext_trait` to be in the crate root

This code:
```rs
use extension_traits::extension;

#[extension(trait Trait)]
impl u32 {
    type Type = ();
}
```
produces this error:
```
error[E0433]: cannot find `ext_trait` in the crate root
 --> src/main.rs:3:1
  |
3 | #[extension(trait Trait)]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^ could not find `ext_trait` in the list of imported crates
  |
  = note: this error originates in the attribute macro `extension` (in Nightly builds, run with -Z macro-backtrace for more info)
```

This can be fixed if `ext-trait` is also included as a dependency, but of course that is not ideal.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -3,7 +3,7 @@ name = "ext-trait"
 authors = [
     "Daniel Henry-Mantilla <daniel.henry.mantilla@gmail.com>"
 ]
-version = "2.0.3-rc1"  # Keep in sync
+version = "0.2.3"  # Keep in sync
 edition = "2018"
 
 license = "Zlib OR MIT OR Apache-2.0"
@@ -22,7 +22,7 @@ ui-tests = ["better-docs"]
 
 [dependencies.ext-trait-proc_macros]
 path = "src/proc_macros"
-version = "2.0.3-rc1"  # Keep in sync
+version = "0.2.3"  # Keep in sync
 
 [dev-dependencies]
 async-trait = "0.1.81"

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -3,7 +3,7 @@ name = "ext-trait"
 authors = [
     "Daniel Henry-Mantilla <daniel.henry.mantilla@gmail.com>"
 ]
-version = "2.0.2"  # Keep in sync
+version = "2.0.3-rc1"  # Keep in sync
 edition = "2018"
 
 license = "Zlib OR MIT OR Apache-2.0"
@@ -22,7 +22,7 @@ ui-tests = ["better-docs"]
 
 [dependencies.ext-trait-proc_macros]
 path = "src/proc_macros"
-version = "2.0.2"  # Keep in sync
+version = "2.0.3-rc1"  # Keep in sync
 
 [dev-dependencies]
 async-trait = "0.1.81"

--- a/cargo_publish.sh
+++ b/cargo_publish.sh
@@ -6,14 +6,6 @@ set -euxo pipefail
     cargo publish
 )
 
-for i in $(seq 10)
-do
-    cargo publish && break
-    sleep 5
-done
+cargo publish
 
-for i in $(seq 10)
-do
-    (cd extension-traits && cargo publish) && break
-    sleep 5
-done
+(cd extension-traits && cargo publish)

--- a/extension-traits/Cargo.toml
+++ b/extension-traits/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "extension-traits"
-version = "2.0.2"  # Keep in sync
+version = "2.0.3-rc1"  # Keep in sync
 authors = ["Daniel Henry-Mantilla <daniel.henry.mantilla@gmail.com>"]
 edition = "2018"
 
@@ -17,7 +17,7 @@ better-docs = []  # allowed to break MSRV
 
 [dependencies.ext-trait]
 path = ".."
-version = "2.0.2"  # Keep in sync
+version = "2.0.3-rc1"  # Keep in sync
 
 [package.metadata.docs.rs]
 all-features = true

--- a/extension-traits/Cargo.toml
+++ b/extension-traits/Cargo.toml
@@ -15,7 +15,8 @@ keywords = ["extension", "trait", "custom", "method", "impl"]
 [features]
 better-docs = []  # allowed to break MSRV
 
-[dependencies.ext-trait]
+[dependencies.ext-trait-renamed]
+package = "ext-trait"
 path = ".."
 version = "2.0.3-rc1"  # Keep in sync
 

--- a/extension-traits/Cargo.toml
+++ b/extension-traits/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "extension-traits"
-version = "2.0.3-rc1"  # Keep in sync
+version = "0.2.3"  # Keep in sync
 authors = ["Daniel Henry-Mantilla <daniel.henry.mantilla@gmail.com>"]
 edition = "2018"
 
@@ -18,7 +18,7 @@ better-docs = []  # allowed to break MSRV
 [dependencies.ext-trait-renamed]
 package = "ext-trait"
 path = ".."
-version = "2.0.3-rc1"  # Keep in sync
+version = "0.2.3"  # Keep in sync
 
 [package.metadata.docs.rs]
 all-features = true

--- a/extension-traits/src/lib.rs
+++ b/extension-traits/src/lib.rs
@@ -5,4 +5,4 @@
 #![forbid(unsafe_code)]
 
 #[doc(inline)]
-pub use ::ext_trait::extension;
+pub use ::ext_trait_renamed::extension;

--- a/extension-traits/tests/regression_issue_12.rs
+++ b/extension-traits/tests/regression_issue_12.rs
@@ -1,0 +1,6 @@
+use extension_traits::extension;
+
+#[extension(trait Trait)]
+impl u32 {
+    type Type = ();
+}

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -34,12 +34,6 @@
 
 pub use ::ext_trait_proc_macros::extension;
 
-// macro internals
-#[doc(hidden)] /** Not part of the public API */ pub
-mod __ {
-    pub use ::core;
-}
-
 #[cfg_attr(feature = "ui-tests",
     cfg_attr(all(), doc = include_str!("compile_fail_tests.md")),
 )]

--- a/src/proc_macros/Cargo.toml
+++ b/src/proc_macros/Cargo.toml
@@ -7,7 +7,7 @@ name = "ext-trait-proc_macros"
 authors = [
     "Daniel Henry-Mantilla <daniel.henry.mantilla@gmail.com>"
 ]
-version = "2.0.3-rc1"  # Keep in sync
+version = "0.2.3"  # Keep in sync
 edition = "2018"
 
 license = "Zlib OR MIT OR Apache-2.0"

--- a/src/proc_macros/Cargo.toml
+++ b/src/proc_macros/Cargo.toml
@@ -7,7 +7,7 @@ name = "ext-trait-proc_macros"
 authors = [
     "Daniel Henry-Mantilla <daniel.henry.mantilla@gmail.com>"
 ]
-version = "2.0.2"  # Keep in sync
+version = "2.0.3-rc1"  # Keep in sync
 edition = "2018"
 
 license = "Zlib OR MIT OR Apache-2.0"

--- a/src/proc_macros/mod.rs
+++ b/src/proc_macros/mod.rs
@@ -183,7 +183,7 @@ fn extension_impl (
                 #default_
                 #type_ #TypeName #generics
                 :
-                    ?::ext_trait::__::core::marker::Sized
+                    ?::core::marker::Sized
                 #SEMICOLON
             )
         },

